src/main/java/blackjack/domain/participant/Participant.java

```diff
@@ -0,0 +1,46 @@
+package blackjack.domain.participant;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardBundle;
+import blackjack.domain.game.ResultType;
+import blackjack.domain.game.Score;
+
+public abstract class Participant {
+
+    protected final String name;
+    protected CardBundle cardBundle;
```

Comment:
```suggestion
    private final String name;
    private final CardBundle cardBundle;
```
해당 상태들은 Participant의 책임이니 private으로 지정하는게 좋을 것 같고, 필요하다면 접근지정자를 protected로 하는게 좋을 것 같아요.

또 CardBundle도 final로 사용할 수 있지 않을까요? 플레이어가 중간에 CardBundle이 재할당되진 않을 것 같아서요.

Reply:
`Player` 와 `Dealer` 의 `receiveCard`, `canReceive`, `toString` 메소드가 각각 다른 로직을 가지고 있어 부득이하게 `cardBundle` (현재 바뀐 이름은 `hand`) 에 접근해야합니다! private 으로 지정하면 각기 다른 로직을 어떻게 처리할 수 있을지 당장은 감이 잡히지 않아요 😢  

cardBundle 은 말씀하신 것 처럼 재할당될 일이 없으므로 final 을 붙여주었습니다 😄

Reply:
오늘 강의를 듣고 감이 잡혔습니다. 추상 클래스의 getter 를 호출해서 필드에 접근하면 되는 문제였는데 괜히 어렵게 생각했나봅니다! ㅎㅎ..